### DISCUSSION TITLE
A topic only shows if there is another topic already selected in search filter

### DISCUSSION THREAD

**Discussion by steveh2023** [microsoft-search/pnp-modern-search#3485] [Q&A]
Hi,

IF rad is typed into the box we get one set of topics - if cancer, another topic, is selected and if rad is then typed into the box another set of topics is shown...

This image shows the issue:

https://ibb.co/J3k8DSB

How can the topics shown be: 1) consistent and 2) full ie all of them....we want all topics starting rad to be shown at all times...

These topics are in the term store...and are being brought in via a refinablestring mapping to a column in the document library called search tags.

**Comment by kasperbolarsen** [comment]
Most likely this behavior is due to we don't get all refiners from the API. Can you check which values are returned using the browser developer tool, filter for postquery

**Reply by wobba** [reply]
Agree with @kasperbolarsen ... once you select and apply one value, then returned values are filtered based on that for subsequent searches. There is no overall state in the component - all API driven.

**Comment by steveh2023** [comment]
Thanks for the suggestions - in the end it appeared the page had been setup incorrectly with 2 search filter web parts - so I removed one and added a 2nd filter under customise filters - edit...
